examples/Recommendations/DismissRecommendation.php

```diff
@@ -98,13 +98,13 @@ public static function main()
      * Runs the example.
      *
      * @param GoogleAdsClient $googleAdsClient the Google Ads API client
-     * @param int $customerId the client customer ID without hyphens
+     * @param int $customerId the customer ID
      * @param string $recommendationId the recommendation ID to dismiss
      */
     public static function runExample(
         GoogleAdsClient $googleAdsClient,
-        $customerId,
-        $recommendationId
+        int $customerId,
+        string $recommendationId
     ) {
         $recommendationResourceName =
             ResourceNames::forRecommendation($customerId, $recommendationId);
```

Comment:
`ResourceNames::forRecommendation` takes an `int` typed variable as second parameter, not a `string`